Kickstarter-iOS/Views/Controllers/ProjectPamphletViewController.swift

```diff
@@ -101,12 +184,57 @@ public final class ProjectPamphletViewController: UIViewController {
     self.viewModel.inputs.willTransition(toNewCollection: newCollection)
   }
 
-  private func setInitial(constraints: [NSLayoutConstraint?], constant: CGFloat) {
+  // MARK: - Private View Setup Functions
+  private func configureSublayers() {
+    if let shapeLayer = self.backThisProjectContainerView.layer.sublayers?.first as? CAShapeLayer {
+      // Shape layer is already added, just needs to be updated with update-to-date bounds
+      let updatedShapeLayer = self.backThisProjectContainerSublayer
+
+      self.backThisProjectContainerView.layer.replaceSublayer(shapeLayer, with: updatedShapeLayer)
+    } else {
+      // Shape layer hasn't been added
+      let maskLayer = self.backThisProjectContainerSublayer
+
+      self.backThisProjectContainerView.layer.insertSublayer(maskLayer, at: 0)
+    }
+  }
 
+  private func setInitial(constraints: [NSLayoutConstraint?], constant: CGFloat) {
     constraints.forEach {
       $0?.constant = constant
     }
   }
+
+  private func goToRewards(project: Project, refTag: RefTag?) {
+
```

Comment:
This could actually be already hooked up to the real VC. Again depending whether we'll be passing a `reward` or not this could look like:

```
private func goToRewards(project: Project, refTag: RefTag?) {
    let leftBarButtonItem = UIBarButtonItem(
      barButtonSystemItem: .cancel,
      target: self,
      action: #selector(dismissPledge)
    )

    let vc = PledgeViewController.instantiate()
    vc.configure(with: project.rewards.first!)
    vc.navigationItem.leftBarButtonItem = leftBarButtonItem

    let nc = UINavigationController(rootViewController: vc)
    self.present(nc, animated: true)
  }
```

and we'd probably want to declare method responsible for dismissal (I'd suggest we do it here since the controller itself should not know how to present/dismiss itself to separate concerns - our app doesn't follow this pattern so it's just a suggestion).

```
@objc func dismissPledge() {
    self.dismiss(animated: true)
  }
```